app/Jobs/TileSingleImage.php

```diff
@@ -61,10 +80,10 @@ public function __construct(Image $image)
     public function handle()
     {
         try {
-            FileCache::getOnce($this->image, [$this, 'generateTiles']);
+            FileCache::getOnce($this->file, [$this, 'generateTiles']);
             $this->uploadToStorage();
-            $this->image->tilingInProgress = false;
-            $this->image->save();
+            $this->file->tilingInProgress = false;
+            $this->file->save();
         } finally {
             File::deleteDirectory($this->tempPath);
```

Comment:
Now I see why the linter complains. You have removed the `File` facade to use the `File` contract from the file cache package. However, the facade is still needed for the call to `File::deleteDirectory()`. Maybe you could add a test case that actually calls the `handle()` method of the job. With such a test, the issue would have become obvious.

Reply:
Thank you for finding that issue!
resolved c5cda584c4a1353dc8ed9945e299ba17dd225417